### PR TITLE
Expose support for filtering aquilon data by user

### PR DESCRIPTION
**Pull request by jrha**
Does not provide a GUI element, but allows the data to be passed through the main page.

### CODE REVIEW

File: views/view-aquilon-sandboxes.php [hunk 1]
```diff
@@ -7,17 +7,29 @@
...
+
+// owner and type for branches
+$url = "$AQUILON_URL/cgi-bin/report/branch_type_owner";
+if ($user) {
+    $url .= "?owner=$user";
+};
+$jsondata = file_get_contents($url);
 $branches = json_decode($jsondata, true);
 
+
```
Comment (tofu-rocketry) [inline views/view-aquilon-sandboxes.php]: Two extra lines of whitespace?

File: views/view-aquilon-sandboxes.php [hunk 2]
```diff
@@ -7,17 +7,29 @@
...
 uksort($all_nodes, "strnatcmp");
 
-$jsondata = file_get_contents("$AQUILON_URL/cgi-bin/report/branch_type_owner");
+
+// owner and type for branches
+$url = "$AQUILON_URL/cgi-bin/report/branch_type_owner";
+if ($user) {
+    $url .= "?owner=$user";
+};
+$jsondata = file_get_contents($url);
```
Comment (tofu-rocketry) [inline views/view-aquilon-sandboxes.php]: Not a fan of all this thrice repeated code.

**Comment by jrha** [comment]
Rebased.

**Comment by jrha** [comment]
@tofu-rocketry now with less duplicated code.

**Review by tofu-rocketry — APPROVED** [review]
Looks ok now, but I'm no PHP expert.

**Pull request merged**